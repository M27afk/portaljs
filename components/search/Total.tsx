export default function Total({ total }) {
  return (
    <h1 className="text-3xl font-semibold text-primary my-6 inline-block">
      { total } results found
    </h1>
  )
}
