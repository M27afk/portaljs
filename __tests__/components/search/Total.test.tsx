import React from 'react'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'
import Total from '../../../components/search/Total'

test('ðŸ“¸ of Total component', ()  => {
  const tree = renderer.create(<Total total={2} />).toJSON()
  expect(tree).toMatchSnapshot()
})
