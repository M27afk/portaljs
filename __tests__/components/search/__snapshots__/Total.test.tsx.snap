// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ“¸ of Total component 1`] = `
<h1
  className="text-4xl font-semibold text-primary"
>
  2
   results found
</h1>
`;
